gui/tools/autorepl.py

```diff
@@ -285,16 +285,16 @@ def write_results():
 
     sshcmd = '%s -p %d %s' % (sshcmd, remote_port, remote)
 
-    remotefs_final = "%s%s%s" % (remotefs, localfs.partition('/')[1], localfs.partition('/')[2])
+    remotefs_final = '%s%s%s' % (remotefs, localfs.partition('/')[1], localfs.partition('/')[2])
 
     # Examine local list of snapshots, then remote snapshots, and determine if there is any work to do.
     log.debug("Checking dataset %s" % (localfs))
 
     # Grab map from local system.
     if recursive:
-        zfsproc = pipeopen('/sbin/zfs list -H -t snapshot -p -o name,creation -r "%s"' % (localfs), debug)
+        zfsproc = pipeopen('/sbin/zfs list -H -t snapshot -p -o name,creation -r \"%s\"' % (localfs), debug)
```

Comment:
Does this change intend to change anything?

Reply:
No, I guess it doesn't change anything. The idea was to be consistent by escaping double-quotes. Do you want me to remove it?

Reply:
Let's make this diff as small as possible please

Reply:
Understood. WIll do!